src/main/java/com/criticalsoftware/Product.java

```diff
@@ -1,23 +1,34 @@
 package com.criticalsoftware;
+
 import org.bson.types.ObjectId;
-import java.awt.image.BufferedImage;
 
 public class Product {
 
     private ObjectId id;
+    private String name;
     private String description;
-    private BufferedImage photo;
+    private String photo;
     private String category;
-    private User user;
+    private User userDonnor;
```

Comment:
Does product need a userDonnor? Who has a userdonnor? The product? The Announcement?